README.md

```diff
@@ -993,6 +999,9 @@ British Machine Vis. Conf. (BMVC), 2018. [YouTube](https://youtu.be/z8uJh_xUq7A)
 - <a name="Rebecq18corl"></a>H. Rebecq, D. Gehrig, D. Scaramuzza,  
 *[ESIM: an Open Event Camera Simulator](http://rpg.ifi.uzh.ch/docs/CORL18_Rebecq.pdf),*  
 Conf. on Robot Learning (CoRL), 2018. [PDF](http://rpg.ifi.uzh.ch/docs/CORL18_Rebecq.pdf), [YouTube](https://youtu.be/ytKOIX_2clo), [Project Page](http://rpg.ifi.uzh.ch/esim/index.html).
+- <a name="chadha2019neuromorphic"></a>Yin Bi, Yiannis Andreopoulos,  
```

Comment:
Could you do the following modifications, to comply with other entries? Thanks.
key: bi19icip
Names: Bi, Y. and Andreopoulos, Y. 
ICIP --> See how other ICIP entries are written in the readme.md file